app/models/feature.rb

```diff
@@ -1,9 +1,12 @@
 class Feature < ApplicationRecord
+  include LockedDocumentConcern
   belongs_to :document
   belongs_to :topical_event
   belongs_to :offsite_link
   belongs_to :feature_list
 
+  before_update :check_if_locked_document_internal
```

Comment:
I think `before_save` might be more appropriate here as we don't want a feature to be created that is associated with a locked document.